### DISCUSSION TITLE
Green LEDs in dark scenes and flickering

### DISCUSSION THREAD

**Discussion by chesterflaps** [awawa-dev/HyperHDR#394] [Q&A]
Hi all.

I am noticing a lot of green light when watching something with a dark scene. Often it will flicker a little as well. Green seems to be the go to colour when it's not sure what to do.

It seems to be worse when watching Doom Patrol but it does have alot of dark scenes.

I am on V16.0.0.1. I'd update to the latest version but do not have a clue how to do it.

**Comment by awawa-dev** [comment]
Hi
Because your version is very old and probably your system glibc version is also out-dated there are 2 paths:
1) Install the system from the scratch, HyperHDR will use default settings but probably it will take only few minutes to write down some important values before beginning from v16 (like LED strip geometry settings) and input them back to v18.
Read more here: https://github.com/awawa-dev/HyperHDR/wiki/Installation#raspberrypi-os-sd-card-image-installation
2) Import your old settings from v16 to v18. That case is more complicated and it should be resolved exactly as updating from v14 to v18.
Read more here: https://github.com/awawa-dev/HyperHDR/discussions/366

If you have your v18 up and running, the next step is to use calibrated LUT for your grabber.
Read more:
https://www.hyperhdr.eu/2022/04/usb-grabbers-hdr-to-sdr-quality-test.html

And here is explained how to use prepared LUT table if your grabber is supported directly.
https://github.com/awawa-dev/HyperHDR/discussions/389

If you have an unsupported USB grabber then you can calibrate it yourself.
https://www.hyperhdr.eu/2022/08/hyperhdr-v18-release.html (point nr 1)

**Reply by chesterflaps** [reply]
Hi. I cannot see on the last link what to actually do to apply the custom LUT. The grabber I have is a Zasluke Game Capture Card which I cannot see on the list of supported grabbers so I clicked your link tor unsupported grabber. I read point 1 and the link in it takes me to the same page as the first link you supplied.

**Reply by awawa-dev** [reply]
Hi
For Zasluke grabber you must have v19beta1 https://github.com/awawa-dev/HyperHDR/releases/tag/v19.0.0.0beta1
For performing LUt calibration please search the forum, or ask a specific question. Each point of the preparation is listed on the HyperHDR LUT calibration page (https://i.ibb.co/42kfPv0/calibration.jpg).

**Reply by chesterflaps** [reply]
Do you mean I must have V19 to use HyperHDR with the grabber in general or do you mean I have to upgrade so I can use the LUTs for that grabber?

I am using the Zasluke currently and it's working, apart from the issues I have raised.

**Reply by awawa-dev** [reply]
In v18 Zasluke grabber doesn't support HDR tone mapping. You have a link to the reported issue in v19beta1 release info.

**Reply by chesterflaps** [reply]
Ok I will look to upgrade to V19 but I was lead to believe upgrading from V18 to V19 was done within HyperHDR but it looks like it's a download
